server/src/services/cleanUserInput.js

```diff
@@ -0,0 +1,8 @@
+export const cleanUserInput = (formInput) => {
+  Object.keys(formInput).forEach((field) => {
+    if (formInput[field] === "") {
```

Comment:
Also, could you eslint-ignore the console statement in Seeder so GitHub stops warning me about it? Thanks!